<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-P39CJOI" failures="0" tests="11" name="PARA_BANK_Runner.PAPABank_Runner" time="39.791" errors="1" timestamp="2023-04-10T17:27:36 IST" skipped="0">
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="repeat" time="1.358"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="Test_with_invalid_Data" time="19.349"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="Test_with_Blank_Data" time="0.712"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="Test_with_Valid_Data" time="1.227"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="Verify_First_Amount" time="0.244"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="create_an_account" time="1.511"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="list_Accounts" time="0.023"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="check_balance" time="1.112"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="get_intial_Amount" time="0.007">
    <error message="empty String" type="java.lang.NumberFormatException">
      <![CDATA[java.lang.NumberFormatException: empty String
at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1842)
at java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
at java.base/java.lang.Double.parseDouble(Double.java:651)
at PARA_BANK_Runner.PAPABank_Runner.get_intial_Amount(PAPABank_Runner.java:118)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- get_intial_Amount -->
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="TranferMoney_and_check" time="13.180"/>
  <system-out/>
  <testcase classname="PARA_BANK_Runner.PAPABank_Runner" name="now_check_Balance_after_Transection" time="1.068"/>
  <system-out/>
</testsuite> <!-- PARA_BANK_Runner.PAPABank_Runner -->
